src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        int indexBall = new Random().nextInt(100);
+        ColorSupplier colorSupplier = new ColorSupplier();
+        return new Ball(colorSupplier.getRandomColor(), indexBall);
```

Comment:
You can directly return the new 'Ball' object without creating a temporary variable 'indexBall'.

Reply:
Fixed